state-manager/Dockerfile

```
@@ -1,14 +1,34 @@
 FROM python:3.12-slim-bookworm
-COPY --from=ghcr.io/astral-sh/uv:latest /uv /uvx /bin/
+
+# Install curl for downloading uv
+RUN apt-get update && apt-get install -y curl ca-certificates \
+    && rm -rf /var/lib/apt/lists/*
+
+# Download uv binary based on target architecture
+ARG TARGETARCH
+RUN case "$TARGETARCH" in \
```

Hadolint (2.12.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)